### PR TITLE
Katas UI review - Multiqubit gates

### PR DESCRIPTION
**Pull request by SoniaLopezBravo**

### CODE REVIEW

File: katas/content/multi_qubit_gates/anti_controlled_gate/solution.md [hunk 1]
```diff
@@ -12,17 +12,17 @@ $$
...
 
 $$
 CNOT = 
  \begin{bmatrix}1 & 0 & 0 & 0\\ 0 & 1 & 0 & 0\\ 0 & 0 & 0 & 1\\ 0 & 0 & 1 & 0 \end{bmatrix}
 $$
 
-We need a way to transform the $CNOT$ gate into the unitary transformation represented by $U$.   
-We remember that the Pauli X gate flips the state in the single-qubit case. Here we need to use a 2-qubit version of this gate, which would affect only the second qubit. We conclude, that the idenity gate needs to be used on the first qubit. In the end, the required gate is a tensor product: $I \otimes X$.
+You need a way to transform the $CNOT$ gate into the unitary transformation represented by $U$.   
+Remember that the Pauli X gate flips the state in the single-qubit case. Here you need to use a 2-qubit version of this gate, which would affect only the second qubit. You conclude, that the idenity gate needs to be used on the first qubit. In the end, the required gate is a tensor product: $I \otimes X$.
```
Comment (tcNickolas) [inline katas/content/multi_qubit_gates/anti_controlled_gate/solution.md:23]: Reads odd, and a typo in an adjacent word
```suggestion
Remember that the Pauli X gate flips the state in the single-qubit case. Here you need to use a 2-qubit version of this gate, which would affect only the second qubit. You can conclude that the identity gate needs to be used on the first qubit. In the end, the required gate is a tensor product: $I \otimes X$.
```

File: katas/content/multi_qubit_gates/entangle_qubits/solution.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-This is exactly the effect of the $CNOT$ gate. Depending upon the state of the **control** qubit (A in our case), the value of the controlled or **target** qubit (B in our case) is inverted or unchanged. Thus, we get the goal state $\alpha \ket{00} + \beta \ket{11}$.  
+Compare the input state $\alpha \ket{0_A0_B} + \beta \ket{1_A0_B}$ with the goal state $\alpha \ket{0_A0_B} + \beta \ket{1_A1_B}$. 
+You want to pass our input qubit through a gate or gates (to be decided) that do the following. If qubit A is in the $\ket{0}$ state, then you want to leave qubit B alone (the first term of the superposition). 
+However, if A is in the $\ket{1}$ state, you want to flip qubit B from $\ket{0}$ into $\ket{1}$ state. In other words, the state of B is to be made contingent upon the state of A. 
+This is exactly the effect of the $CNOT$ gate. Depending upon the state of the **control** qubit (A in your case), the value of the controlled or **target** qubit (B in your case) is inverted or unchanged. Thus, you get the goal state $\alpha \ket{00} + \beta \ket{11}$.
```
Comment (tcNickolas) [inline katas/content/multi_qubit_gates/entangle_qubits/solution.md:6]:
```suggestion
This is exactly the effect of the $CNOT$ gate. Depending upon the state of the **control** qubit (A in this case), the value of the controlled or **target** qubit (B in this case) is inverted or unchanged. Thus, you get the goal state $\alpha \ket{00} + \beta \ket{11}$.  
```

File: katas/content/multi_qubit_gates/fredkin_gate/solution.md [hunk 3]
```diff
@@ -12,13 +12,13 @@ $$
 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1
 \end{bmatrix}
 $$
-and our initial state is:
+and your initial state is:
```
Comment (tcNickolas) [inline katas/content/multi_qubit_gates/fredkin_gate/solution.md:15]:
```suggestion
and the initial state is:
```

File: katas/content/multi_qubit_gates/toffoli_gate/solution.md [hunk 4]
```diff
@@ -11,13 +11,13 @@ $$
  0 & 0 & 0 & 0 & 0 & 0 & 1 & 0
  \end{bmatrix}
 $$
-and our initial state is:
+and your initial state is:
```
Comment (tcNickolas) [inline katas/content/multi_qubit_gates/toffoli_gate/solution.md:14]:
```suggestion
and the initial state is:
```

File: katas/content/multi_qubit_gates/index.md [hunk 5]
```diff
@@ -81,12 +83,12 @@ It can be less straightforward when a multi-qubit gate is applied to a subset of
     "title": "CNOT Gate"
 })
 
-Our first proper multi-qubit gate is the $CNOT$ ("controlled NOT") gate. The $CNOT$ gate is a two-qubit gate, with one qubit referred to as the **control** qubit, and the other qubit as the **target** qubit (usually the first qubit is the control, and the second qubit is the target).
+The first multi-qubit gate you're going to review is the $CNOT$ ("controlled NOT") gate. The $CNOT$ gate is a two-qubit gate, with one qubit referred to as the **control** qubit, and the other qubit as the **target** qubit (usually the first qubit is the control, and the second qubit is the target).
```
Comment (tcNickolas) [inline katas/content/multi_qubit_gates/index.md:86]:
```suggestion
The first multi-qubit gate this kata introduces is the $CNOT$ ("controlled NOT") gate. The $CNOT$ gate is a two-qubit gate, with one qubit referred to as the **control** qubit, and the other qubit as the **target** qubit (usually the first qubit is the control, and the second qubit is the target).
```

**Review by tcNickolas — APPROVED** [review]
Looks good, several small comments

File: katas/content/multi_qubit_gates/anti_controlled_gate/solution.md [hunk 6]
```diff
@@ -12,17 +12,17 @@ $$
 U = \begin{bmatrix}0 & 1 & 0 & 0\\ 1 & 0 & 0 & 0\\ 0 & 0 & 1 & 0\\ 0 & 0 & 0 & 1 \end{bmatrix}
 $$
 
-We remember a two-qubit gate with a similar matrix representation - the $CNOT$ gate:
+You remember a two-qubit gate with a similar matrix representation - the $CNOT$ gate:
```
Comment (cesarzc) [inline katas/content/multi_qubit_gates/anti_controlled_gate/solution.md:15]:
```suggestion
Remember a two-qubit gate with a similar matrix representation - the $CNOT$ gate:
```

Comment (cesarzc) [reply katas/content/multi_qubit_gates/anti_controlled_gate/solution.md:15]: I think in this case it makes sense to drop the "You".

File: katas/content/multi_qubit_gates/index.md [hunk 7]
```diff
@@ -224,7 +226,7 @@ $$=\ket{00}\braket{00|10} + \ket{01}\braket{01|10} + \ket{10}\braket{11|10} + \k
 $$=\ket{00}\big(\braket{00|10}\big) + \ket{01}\big(\braket{01|10}\big) + \ket{10}\big(\braket{11|10}\big) + \ket{11}\big(\braket{10|10}\big) =$$
 $$=\ket{00}(0) + \ket{01}(0) + \ket{10}(0) + \ket{11}(1) = \ket{11}$$
 
-> Notice how a lot of the inner product terms turn out to equal 0, and our expression is easily simplified. We have expressed the $CNOT$ gate in terms of outer product of computational basis states, which are orthonormal, and apply it to another computational basis state, so the individual inner products are going to always be 0 or 1.
+> Notice how a lot of the inner product terms turn out to equal 0, and the expression is easily simplified. The $CNOT$ gate is expressed in terms of outer product of computational basis states, which are orthonormal, and is applied to another computational basis state, so the individual inner products are going to always be 0 or 1.
```
Comment (cesarzc) [inline katas/content/multi_qubit_gates/index.md:229]:
```suggestion
> Notice how a lot of the inner product terms turn out to equal 0, and the expression is easily simplified. The $CNOT$ gate is expressed in terms of the outer product of computational basis states, which are orthonormal, and is applied to another computational basis state, so the individual inner products will always be 0 or 1.
```

**Review by cesarzc — APPROVED** [review]
Pointed out a few minor things.

**Pull request merged**